change/@microsoft-teams-js-1db265e1-1b99-420b-b091-137d838d3982.json

```diff
@@ -0,0 +1,7 @@
+{
+  "type": "none",
+  "comment": "Update interfaces.ts",
```

Comment:
>Update interfaces.ts [](http://example.com/codeflow?start=14&length=20)

Since this message appears in the public CHANGELOG, can you update it to be more descriptive? E.g. "Clarified documentation for `sessionId` and `appSessionId` context properties"

Reply:
I don't know how to change this now. I did it initially when I had to add a changefile.